translations/en.json

```diff
@@ -34,7 +34,7 @@
     "add-network": "Add network",
     "add-node": "Add node",
     "add-to-contacts": "Add to contacts",
-    "add-to-contacts-text": "By adding a user to your contact list, you share your wallet address",
+    "add-to-contacts-text": "By adding a user to your contact list, you share your wallet address",
```

Comment:
What's changing here? Seems like the same text or I am not being able to figure it out, maybe the diff is a different space character. Same apply to other texts below

Reply:
I think that change was make via `make lint-fix`

Reply:
Hmmm that's not happening to me on latest `develop` branch, however I've communicated the issue internally just in case. Can you revert these changes if they are not related to the purpose of this PR?

Reply:
Will revert the changes then!